### PR TITLE
#157627657 remodel UI design on the request page and landing pages

### PR DESCRIPTION
**Pull request by PascalUlor**
#### What does this PR do?
To implement the necessary feedback on the UI Templates

#### Description of Task to be completed?
Used flexbox to align the Nav-bar and the request list

#### How should this be manually tested?
1. clone to the local machine
2. cd into the folder
3. open in the browser. (e.g. Chrome)

#### What are the relevant pivotal tracker stories?
[157627657](https://www.pivotaltracker.com/story/show/157627657)

## Previously 
image

image

image

## Presently
image

image

image

image

### CODE REVIEW

**Comment by benfluleck** [comment]
Make sure you re-read your PRS before putting them up, there are a few grammatical errors in the PR body

File: UI/public/css/main.css [hunk 1]
```diff
@@ -0,0 +1,250 @@
+  /* Nav and Footer Styling */
+
+  * {
+  	box-sizing: border-box;
+  }
+
+  body {
+  	font-family: 'Monserrat', sans-serif;
```
Comment (benfluleck) [inline UI/public/css/main.css:11]: These are not indented properly, also the braces should align with each other. Additionally, I thought we discussed the introduction of google font.

File: UI/public/fonts/Anton/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+DEFINITIONS
+"Font Software" refers to the set of files released by the Copyright
+Holder(s) under this license and clearly marked as such. This may
+include source files, build scripts and documentation.
+
+"Reserved Font Name" refers to any names specified as such after the
+copyright statement(s).
+
+"Original Version" refers to the collection of Font Software components as
```
Comment (benfluleck) [inline UI/public/fonts/Anton/OFL.txt]: Don't install extra fonts on your site as it takes away space you could use for other things, rather use a CDN to import it.

File: UI/public/css/main.css (hunk 1 above)
Comment (PascalUlor) [reply UI/public/css/main.css:11]: I used google font on the form.css and request.css files

File: UI/public/fonts/Anton/OFL.txt (hunk 2 above)
Comment (PascalUlor) [reply UI/public/fonts/Anton/OFL.txt]: ok I'll take them out

File: UI/public/css/main.css (hunk 1 above)
Comment (PascalUlor) [reply UI/public/css/main.css:11]: I have added the cdn and deleted the fonts folder...

**Pull request merged**